README.md

```diff
@@ -56,6 +56,12 @@ subscribed RHEL machine.
     $ make build TARGET=rhel7 VERSION=2
     ```
 
+You can also run the build using Podman. However, e2e tests does not run  
+with Podman (which is the default).
+    ```
+    $ make build TARGET=rhel7 VERSION=2 BUILT_COMMAND="podman build"
```

Comment:
looks like it is `BUILD_COMMAND` below, not `BUILT_COMMAND`

Reply:
good catch, I will push it in joint PR thanks @gabemontero